vendor/bat-native-ledger/src/bat/ledger/internal/gemini/gemini_transfer.cc

```diff
@@ -101,11 +102,13 @@ void GeminiTransfer::OnTransactionStatus(const type::Result result,
                                          const int attempts,
                                          client::TransactionCallback callback) {
   if (result == type::Result::LEDGER_OK) {
+    retry_timer_.erase(id);
```

Comment:
Should we just always call `erase` as the first thing that this callback does?

Reply:
resolved!